code/src/components/AddTask.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { useDispatch } from 'react-redux'
+
+import { tasks } from 'reducers/tasks'
+
+export const AddTask = () => {
+  const dispatch = useDispatch()
+  //State for input from textbox
+  const [value, setValue] = useState("")
+
+
+  //Handle submit function to dispatch task
+  const handleSubmit = (event) => {
+    event.preventDefault()
+
+    //Dispatch the action to save the new task.
+    dispatch(
+      tasks.actions.addTask({
+        text: value,
+        done: false
+      })
+    )
+
+    //Clear the text field
+    setValue("")
+  }
+
+  return (
+    <>
+      <Form onSubmit={handleSubmit}>
+        <InputArea
+          type="text"
+          placeholder="Add task"
+          onChange={event => setValue(event.target.value)}
+          value={value}>
+        </InputArea>
+        <AddButton
+          type="submit"><span role="img" aria-label="Add">➕</span></AddButton>
+      </Form>
+    </>
+  )
+}
+
+const Form = styled.form`
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+`
+const InputArea = styled.input`
+  height: 30px;
+`
+const AddButton = styled.button`
+  background-color: #e6e6e6; 
+  padding: 5px;
+  border-radius: 50%;
+  height: 30px;
+  width: 30px;
+  align-items: center;
+  justify-content: center;
```

Comment:
I think you need to add `display:flex;` to be able to make use of these. I tried adding it in the developer tools, and it did center the plus-sign within the button. 😊